backend/routes/projects.js

```diff
@@ -30,4 +52,63 @@ router.get("/", [authorizeUser(["roster"])], (req, res, next) => {
       });
 });
 
+/**
+ * Returns a specific project given its id.
+ */
+router.get("/:project_id", [authorizeUser(["roster"])], (req, res, next) => {
+  getOneProject(req.params.project_id)
+    .then((project) => {
+      if (project === null)
+        res.status(400).json({ message: "Project with the given id doesn't exist" });
+      else res.status(200).json({ project });
+    })
+    .catch((err) => {
+      next(err);
+    });
+});
+
+/**
+ * Creates a new project given that the user has permissions to
+ * create projects.
+ */
+router.post(
+  "/",
+  [
+    validators.name,
+    validators.description,
+    validators.client,
+    validators.phase,
+    validators.timeline_quarter,
+    validators.timeline_year,
+    validators.project_manager,
+    validators.outreach,
+    validators.designers,
+    validators.developers,
+    validators.file_link,
+    validators.file_name,
+    validateRequest,
+    authorizeUser(["roster", "project_create"]),
+  ],
+  (req, res, next) => {
+    addProject({
+      name: req.body.name,
+      description: req.body.description,
+      client: req.body.client,
+      phase: req.body.phase,
+      timeline: req.body.timeline,
+      project_manager: req.body.project_manager,
+      outreach: req.body.outreach,
+      designers: req.body.designers,
+      developers: req.body.developers,
+      files: req.body.files,
+    })
+      .then((project) => {
+        res.status(200).json({ project });
```

Comment:
```suggestion
        res.status(200).json(project);
```

Reply:
Sorry if I was unclear earlier - the other routes return the created object directly :sweat_smile:

Reply:
which routes are you referring to here because all of the other ones that I saw encapsulate it in a JSON?

Reply:
Oh sorry, you're right! I was looking at the in the services directory when I should've been looking in the routes directory. My bad :sweat_smile: